frontend/src/ScoreBox.js

```diff
@@ -298,29 +297,31 @@ class ScoreBox extends Component {
       width: 270,
       show: true,
       required: true,
-      Cell: props => (
+      Cell: ({ original, value }) => (
```

Comment:
Would prefer further destructuring (like getting key and star out of original) and also updating the other Cell component above to match this destructuring. Also, can you add default values?